tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py

```diff
@@ -448,8 +448,7 @@
         Field(
             item="67",
             name="DATE_OF_BIRTH",
-            friendly_name="date of birth",
-            type="string",
```

Comment:
@victoriaatraft looks like the type= line got dropped here, can you add `type="string",` back in alongside the other final tweaks? That should resolve the build errors.